README.md

```diff
@@ -9,11 +9,17 @@ During the transition to the new folder structured in 2.6.5 required by the plug
 - Run the script `./scripts/rejig` to move folders to their new location in the `shell` folder and update the appropriate import statements
   Use this to convert older PRs to the new format
 - Run the script `./scripts/rejig -d` to move folders to their old location and update imports again
-  Use this to convert newer branches to the old format (possibly useful for branches) 
+  Use this to convert newer branches to the old format (possibly useful for branches)
 
 For more information on plugins see [Plugins](./docs/developer/PLUGINS.md).
 
 ## Running for Development
+Create an `.env` file, this will allow us to specify a Rancher API endpoint and exclude different packages.
```

Comment:
I think he wanted to make sure everyone knows it's possible to set a bunch of envs like this.
I agree with you about moving the plugin part in the same file.

Reply:
Any suggestions on where it could fit?  Not sure if it has to be duplicated per Use case or have a new section.

Reply:
I think it may make sense to refer to the setup and template with a link but for each section mention their own env var. What do you think? @richard-cox

Reply:
This would fit in to https://github.com/rancher/dashboard/blob/master/docs/developer/PLUGINS.md#use-case-dynamically-loading-a-ui-plugin where `excludes` is discussed.

If there needs to be a specific page for the file it should fit in to https://github.com/rancher/dashboard/tree/master/docs/developer. Would suggest it contains a table containing each env the dashboard supports (see usages of `process.env.`, or state that there's more envs to follow)

Reply:
Added it to the file.